app/controllers/webhooks_controller.rb

```diff
@@ -36,4 +36,27 @@ def gocardless
 
     head(:ok)
   end
+
+  def adyen
+    signature = adyen_params.dig("additionalData", "hmacSignature")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)